libvips/resample/templates.h

```diff
@@ -444,7 +444,24 @@ calculate_coefficients_lanczos( double *c,
  */
 template <typename T, typename IT>
 static IT
-reduce_sum( const T * restrict in, int stride, const IT * restrict c, int n )
+reduce_sum( const T * restrict in, int stride, const int * restrict c, int n,
```

Comment:
Do we need to `#include <type_traits>`? (I feel like this is the sort of thing that MSVC would complain about, maybe one day I'll get round to adding it to CI.)

Reply:
Good point, I added `#include <type_traits>` on the top of this file to ensure compat with MSVC.